File: kildey004/programacaobasica/Questao3.java
import java.util.Scanner;
public class Questao3 {
    public static void main (String[] args) {
        int nc;
        int ncd ;
        int ctotal;
        int cdtotal;
        int total;
        Scanner s;

        System.out.println("**dinheiro recebido**");
        s = new Scanner(System.in);
        System.out.println("Digite a quantidade de casamentos: ");
        nc = s.nextInt();

        System.out.println("Digite a quantidade de certidoes: ");
        ncd = s.nextInt();

        ctotal = nc * 250;
        cdtotal = ncd * 30;

        total = ctotal + cdtotal;

        System.out.println("Total ganho em certidoes: R$" + cdtotal);
        System.out.println("Total ganho em casamentos: R$" + ctotal);
        System.out.println("Ganho total: R$" + total);
    }
}

File: kildey004/programacaobasica/questao3.py
print (" ******************************** ")
print (" **Calculo de dinheiro recebido** ")
print (" ******************************** ")
casamento = int (250)
certidao =int (30)

nC = int (input(" Digite a quantidade de casamentos: "))
nCD = int (input(" Digite a quantidade de certidoes: "))

Ctotal = casamento * nC
CDtotal = certidao * nCD

total = Ctotal + CDtotal

print(" Total ganho com casamentos: R$" + str(Ctotal))
print(" Total ganho com certidoes: R$" + str(CDtotal))
print(" Total geral: R$" + str(total))



